src/content/staking/solo/index.md

```diff
@@ -0,0 +1,175 @@
+    When operating your own node you should spend some time learning how to use the software you've chosen. This should involve reading relevant documentation and being attune to communication channels of those dev teams.
+    The more you understand about the software you're running and how proof-of-stake works, the less risky it will be as a staker, and the easier it will be to fix any issues that may arise along the way as a node operator. 
+  </ExpandableCard>
+  <ExpandableCard title="Comfortable with computers" eventCategory="SoloStaking" eventName="clicked comfortable with computers">
+    Node setup requires a reasonable comfort level when working with computers, although new tools are making this easier over time. Understanding of the command-line interface is helpful, but no longer strictly required 
+    It also requires very basic hardware setup, and some understanding of minimum recommended specs.
+  </ExpandableCard>
+  <ExpandableCard title="Secure key management" eventCategory="SoloStaking" eventName="clicked secure key management">
+    Just like how private keys secure your Ethereum address, you will need to generate keys specifically for your validator. You must understand how to keep any seed phrases or private keys safe and secure.
+    <div style={{marginTop: "1rem"}}><ButtonLink to="/security">Ethereum security and scam prevention</ButtonLink></div>
+  </ExpandableCard>
+  <ExpandableCard title="No withdrawing (for now)" eventCategory="SoloStaking" eventName="clicked no withdrawing">
+    Withdrawing staked ETH or rewards from a validator balance is not yet supported. Support for withdrawals are planned for the Shanghai upgrade following The Merge. You should anticipate your ETH being locked for at least one-to-two years. After Shanghai, this will no longer be the case, at which point you will be able to freely withdraw portions or all of your stake if you wish.
+  </ExpandableCard>
+  <ExpandableCard title="Maintenance" eventCategory="SoloStaking" eventName="clicked maintenance">
+    Hardware occasionally fails, connections error out, and client software occasionally needs upgrading. Node maintenance is inevitable and will occasionally require your attention. You'll want to be sure you stay aware of any anticipated network upgrades, or other critical client upgrades.
+  </ExpandableCard>
+  <ExpandableCard title="Reliable uptime" eventCategory="SoloStaking" eventName="clicked reliable uptime">
+    Your rewards are proportional to the time your validator is online and properly attesting. Downtime incurs penalties roughly equal to what you would have earned in that same amount of time. Being offline while the network is finalizing does not result in slashing.
+    Epoch rewards are inversely proportional to how long it takes your attestations to be included, meaning internet bandwidth also matter. This can vary, but a minimum of 10 Mb/s up and down is recommended.
```

Comment:
> Being offline while the network is finalizing does not result in slashing. Epoch rewards are inversely proportional to how long it takes your attestations to be included, meaning internet bandwidth also matter. 

This confused me. I wonder if we can reword or drop this? Perhaps we link to a source or get into the nitty-gritty of how rewards & penalties are calculated elsewhere?

Reply:
Will reword this